package.json

```diff
@@ -20,20 +20,17 @@
     }
   },
   "browser": "./main.js",
```

Comment:
We should remove this line as o-brand has no javascript

Reply:
```suggestion
```